certified-connectors/Muhimbi PDF Converter Online/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4352 @@
+          "type": "string",
+          "format": "enum",
+          "default": "Both",
+          "uniqueItems": false,
+          "enum": [
+            "Portrait",
+            "Landscape",
+            "Both"
+          ]
+        },
+        "print_only": {
+          "description": "Tells if the watermark is only visible when the document is printed",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Print only",
+          "type": "string",
+          "format": "enum",
+          "default": "false",
+          "uniqueItems": false,
+          "enum": [
+            "false",
+            "true"
+          ]
+        },
+        "start_section": {
+          "description": "The first section in a Word or Excel document the watermark is visible",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Watermark start section",
+          "type": "integer",
+          "format": "int32",
+          "uniqueItems": false
+        },
+        "end_section": {
+          "description": "The last section in a Word or Excel document the watermark is visible",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Watermark end section",
+          "type": "integer",
+          "format": "int32",
+          "uniqueItems": false
+        },
+        "page_type": {
+          "description": "One or more page types for Word and Excel documents, eg. default|first|even",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Watermark page type",
+          "type": "string",
+          "uniqueItems": false
+        },
+        "fail_on_error": {
+          "description": "Fail on error",
+          "x-ms-visibility": "advanced",
+          "x-ms-summary": "Fail on error",
+          "type": "boolean",
+          "default": true,
+          "uniqueItems": false
+        }
+      },
+      "type": "object",
+      "format": "text_watermark_data"
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://api.muhimbi.com/auth/pages/authorize.aspx",
+      "tokenUrl": "https://api.muhimbi.com/auth/pages/gettoken.aspx?grant_type=authorization_code",
+      "scopes": {
+        "Flow": "Flow"
+      }
+    },
+    "api_key": {
+      "type": "apiKey",
+      "name": "api_key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "oauth2_auth": [
+        "Flow"
+      ]
+    }
+  ],
+  "tags": [],
+  "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": " http://www.muhimbi-online.com/ "
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://support.muhimbi.com/hc/en-us/articles/228090747-Muhimbi-Ltd-Privacy-Policy"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "Collaboration;Content and Files; "
```

Comment:
@mmohapatra-muhimbi Please remove the semicolon and space at the end of the property value of "Categories" like below as It would be syntax error.
`"propertyValue": "Collaboration;Content and Files"`

Reply:
@sriyen-msft , modified the swagger file as per above suggestion did git commit and git push, do i have to do pull out request again?